_posts/pandera-dataframe-validation.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: single
+title: "Statistical Data Validation with Pandera"
```

Comment:
```suggestion
title: "Statistical Data Validation on Python Pandas DataFrames with Pandera"
```

Reply:
very open to suggestions but i'd just like to see a few other words in the title like Python and Pandas. feel free to move words around!